composer/spec/fixtures/composer_files/env_variable_params

```diff
@@ -0,0 +1,28 @@
+{
+  "repositories": [
+    {
+      "type": "composer",
+      "url": "https://wpackagist.org"
+    },
+    {
+      "type": "package",
+      "package": {
+        "name": "advanced-custom-fields/advanced-custom-fields-pro",
+        "version": "5.5.14",
+        "type": "wordpress-plugin",
+        "dist": {
+          "type": "zip",
+          "url": "https://connect.advancedcustomfields.com/index.php?a=download&p=pro&k={%ACF_PRO_KEY}"
```

Comment:
Hmm not sure what is going on here, looks like this fails unless the `p=pro` comes first, `https://connect.advancedcustomfields.com/index.php?p=pro&a=download&k={%ACF_PRO_KEY}` but this seems to error with the existing error message: `Could not find a key for ACF PRO`.

Just responded here https://github.com/dependabot/feedback/issues/714#issuecomment-545033193 as I was getting a bunch of errors locally complaining about the `version` environment variable. Not sure if we should be supporting it somehow.